cluster/gce/config-default.sh

```diff
@@ -77,6 +77,12 @@ DNS_SERVER_IP="10.0.0.10"
 DNS_DOMAIN="cluster.local"
 DNS_REPLICAS=1
 
+# Optional: Install cluster docker registry.
+ENABLE_CLUSTER_REGISTRY="${KUBE_ENABLE_CLUSTER_REGISTRY:-true}"
+CLUSTER_REGISTRY_DISK="${CLUSTER_REGISTRY_PD:-kube-system-kube-registry}"
```

Comment:
I think a sane default here is something based on the `INSTANCE_PREFIX` for multi-cluster environments (so people don't have to twiddle a thousand things)